transaction/builder/src/transaction_builder.rs

```diff
@@ -3247,29 +3083,14 @@ pub mod transaction_builder_tests {
             let mut memo_builder = BurnRedemptionMemoBuilder::new([2u8; 64]);
             memo_builder.enable_destination_memo();
 
-            let mut transaction_builder = TransactionBuilder::new(
-                block_version,
-                Amount::new(10, token_id),
-                fog_resolver.clone(),
-            )
-            .unwrap();
-
+            let mut transaction_builder = transaction_builder.clone();
             transaction_builder.set_fee(3).unwrap();
 
-            let input_credentials = get_input_credentials(
-                block_version,
-                Amount::new(113, token_id),
-                &AccountKey::random(&mut rng),
-                &fog_resolver,
-                &mut rng,
-            );
-            transaction_builder.add_input(input_credentials);
-
             let TxOutContext {
                 tx_out_public_key: burn_output_public_key,
                 ..
             } = transaction_builder
-                .add_output(Amount::new(110, token_id), &burn_address(), &mut rng)
+                .add_output(Amount::new(497, token_id), &burn_address(), &mut rng)
```

Comment:
Here (and in other places below), the amount got updated so that the transaction math is valid.